inc/class-command.php

```diff
@@ -187,6 +187,19 @@ public function check( $args, $assoc_args ) {
 
 		$formatter = new Formatter( $assoc_args, array( 'name', 'status', 'message' ) );
 		$formatter->display_items( $results );
+
+		$results_with_error = array_filter( $results, function( $check ){
+			return 'error' === $check['status'];
+		});
+		if ( ! empty( $results_with_error ) ) {
+			if ( 'table' === $assoc_args['format'] ) {
+				$check_count = count( $results_with_error );
+				$message = 1 === $check_count ? "1 check reports 'error'." : "${check_count} checks report 'error'.";
```

Comment:
Can we update the tests to include coverage for this output?

Reply:
Yeah, I had some trouble with that for some reason. But I probably figured a few things out since then so I'll try again as soon as I can.

Reply:
@danielbachhuber just a heads up -- the issue was that `Formatter` removes  `$assoc_args['format']`, so I had to change it do that check before formatter runs. Don't know if there's a specific reason it works that way, but it caught me off guard.
https://github.com/wp-cli/wp-cli/blob/master/php/WP_CLI/Formatter.php#L37